tests/xfstests/install.pm

```diff
@@ -42,7 +42,8 @@ sub collect_version {
 
 sub run {
     my $self = shift;
-    $self->select_serial_terminal;
+    # Use root-console for installation on svirt instead of root-sut-serial
+    check_var('BACKEND', 'svirt') ? select_console('root-console') : $self->select_serial_terminal;
```

Comment:
the reason why there is a helper method is exactly to avoid needing to do any backend specific checks here. That there even *is* a "select_serial_terminal" method within tests is a stretch already. Please do not introduce backend specific handling in test modules.

Reply:
I could find only passed example is https://openqa.suse.de/tests/3591518 . So I follow the same way to solve it. Why select_serial_terminal not work in svirt, and in my VR, it add some strange console/*** tests in the end. I didn't found any hint in main.pm . I should put a WIP in this PR.

Reply:
Good that are you are looking into this. Appreciated.